podpac/core/data/ogc.py

```diff
@@ -4,373 +4,264 @@
-            capabilities = r.data
-            if r.status != 200:
-                raise Exception("Could not get capabilities from WCS server:" + self.capabilities_url)
-        else:
-            raise Exception("Do not have a URL request library to get WCS data.")
+            def selector(rsc, rsci, coordinates):
+                return coordinates, tuple(slice(None) for dim in coordinates)
 
-        if (
-            lxml is not None
-        ):  # could skip using lxml and always use html.parser instead, which seems to work but lxml might be faster
-            capabilities = bs4.BeautifulSoup(capabilities, "lxml")
-        else:
-            capabilities = bs4.BeautifulSoup(capabilities, "html.parser")
-
-        domain = capabilities.find("wcs:spatialdomain")
-        pos = domain.find("gml:envelope").get_text().split()
-        lonlat = np.array(pos, float).reshape(2, 2)
-        grid_env = domain.find("gml:gridenvelope")
-        low = np.array(grid_env.find("gml:low").text.split(), int)
-        high = np.array(grid_env.find("gml:high").text.split(), int)
-        size = high - low
-        dlondlat = (lonlat[1, :] - lonlat[0, :]) / size
-        bottom = lonlat[:, 1].min() + dlondlat[1] / 2
-        top = lonlat[:, 1].max() - dlondlat[1] / 2
-        left = lonlat[:, 0].min() + dlondlat[0] / 2
-        right = lonlat[:, 0].max() - dlondlat[0] / 2
-
-        timedomain = capabilities.find("wcs:temporaldomain")
-        if timedomain is None:
-            return Coordinates(
-                [
-                    UniformCoordinates1d(top, bottom, size=size[1], name="lat"),
-                    UniformCoordinates1d(left, right, size=size[0], name="lon"),
-                ]
-            )
-
-        date_re = re.compile("[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}")
-        times = str(timedomain).replace("<gml:timeposition>", "").replace("</gml:timeposition>", "").split("\n")
-        times = np.array([t for t in times if date_re.match(t)], np.datetime64)
-
-        if len(times) == 0:
-            return Coordinates(
-                [
-                    UniformCoordinates1d(top, bottom, size=size[1], name="lat"),
-                    UniformCoordinates1d(left, right, size=size[0], name="lon"),
-                ]
-            )
-
-        return Coordinates(
-            [
-                ArrayCoordinates1d(times, name="time"),
-                UniformCoordinates1d(top, bottom, size=size[1], name="lat"),
-                UniformCoordinates1d(left, right, size=size[0], name="lon"),
-            ]
-        )
+            return super()._eval(coordinates, output=output, _selector=selector)
 
-    @common_doc(COMMON_DATA_DOC)
-    def get_coordinates(self):
-        """{get_coordinates}
+        # for uniform stacked, unstack to use the requested coordinates (the WCS server will interpolate)
+        if "lat_lon" in coordinates.dims and coordinates["lat"].is_uniform and coordinates["lon"].is_uniform:
```

Comment:
what about `"lon_lat"`, or `"lon_time_lat"`, or ... 

in my branch I defined the `"is_stacked"` helper: 
```python
if coordinates.is_stacked('lat') and coordinates.is_stacked('lon') and coordinates["lat"].is_uniform and coordinates["lon"].is_uniform```

Reply:
Yeah, thanks. Fixed now.